Resources/Prototypes/Procedural/salvage_loot.yml

```diff
@@ -5,6 +5,30 @@
   loots:
     - !type:RandomSpawnsLoot
       entries:
+        #Custom shit, feel free to change up the values and remove stuff
```

Comment:
you should put this below the regular spawn table and also some of these need to be much rarer or its going to cheapen it way too much